assembl/static2/js/app/components/debate/brightMirror/fictionComment.jsx

```diff
@@ -174,19 +179,38 @@ export class FictionComment extends Component<LocalFictionCommentProps, FictionC
       circleAvatar,
       children,
       commentContent,
-      commentParentId,
+      commentId,
+      contentLocale,
       displayedPublishedDate,
       numChildren,
       modified,
       parentPostAuthorFullname,
+      publicationState,
       publishedDate,
       fictionCommentExtraProps
     } = this.props;
     const { isEditing, showFictionCommentForm, updatedCommentContent, updatedModified } = this.state;
     const { expandedFromTree, expandCollapseCallbackFromTree } = fictionCommentExtraProps;
 
+    // Translation key
+    const postEditedMsgKey = 'debate.thread.postEdited';
+    const deleteCommentBodyMessageMsgKey = 'debate.brightMirror.commentFiction.deleteCommentBodyMessage';
+    const numberOfResponsesMsgKey = 'debate.brightMirror.numberOfResponses';
+
+    // Display DeletedFictionComment component when comment is marked as DELETED_BY_USER or DELETED_BY_ADMIN
```

Comment:
I don't see any use of the ```DELETED_BY_ADMIN``` you mention in the comment. Down in the component you simply display it as deleted by admin if !deleted_by_user ?

Reply:
I just added a new comment to lift the ambiguity:
`// isDeletedByAuthor is true if DELETED_BY_USER, is false if DELETED_BY_ADMIN` because of `publicationState in DeletedPublicationStates` and `DeletedPublicationStates` can only hold DELETED_BY_USER | DELETED_BY_ADMIN. Hope that is clear.

Reply:
It is, thank you very much Sir ! 🙏